style.css

```diff
@@ -412,3 +448,230 @@ ul .small-button-wrapper {
   margin-bottom: 0;
   font-size: 15px;
 }
+
+@media screen and (min-width: 992px) {
```

Comment:
I suggest you increase the min-width as the previous reviewer asked to eg (min-width: 993 - upward) min-width should be greater than 992px to 1100px as this is the required media query for desktop applications

Reply:
nice job 👍